sdk/core/azure_core/benches/core_benchmarks.rs

```diff
@@ -0,0 +1,31 @@
+use azure_core::http::Url;
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use std::time::Duration;
+// how many params to add to the url
+const PARAMS: &[usize] = &[1, 10, 100, 1000];
+
+fn url_parsing_benchmark(c: &mut Criterion) {
+    let mut group = c.benchmark_group("url_parsing");
```

Comment:
Why'd you put this in a group? Devs don't need groups by default. I did it in <https://github.com/heaths/ordinal-rs/blob/main/benches/ordinal.rs> because I'm actually trying to group different crates' benchmarks together, as you show up in <https://github.com/heaths/ordinal-rs/blob/main/README.md#performance>. In general, you don't need this. We want initial examples to be the golden path, and groups aren't it.

Reply:
done